### DISCUSSION TITLE
ValueError: No available memory for the cache blocks. Try increasing `gpu_memory_utilization` when initializing the engine.

### DISCUSSION THREAD

**Discussion by ooooma** [netease-youdao/QAnything#294] [Q&A]
Can any one tell me how to change the `gpu_memory_utilization` when initializing the engine.

**Comment by TatyOko28** [comment]
The error ValueError: No available memory for the cache blocks. Try increasing gpu_memory_utilization indicates that the GPU's memory allocation is insufficient for initializing the engine. You can resolve this by increasing the gpu_memory_utilization parameter during the initialization process.

Here’s how to adjust it:
